user/notifications.md

```diff
@@ -495,6 +496,14 @@ You can specify multiple channels as well.
         on_failure: [always|never|change] # default: always
         on_start: [always|never|change]   # default: never
 
+You also have the possibility to customize the message that will be sent to the channel(s) with a template:
+
+    notifications:
+      slack:
+        template:
+          - 'Build <%{build_url}|#%{build_number}> (<%{compare_url}|%{commit}>) of %{repository}@%{branch} by %{author} %{result} in %{elapsed_time}'
+          - '%{pull_request_url} %{commit_subject}'
+
 As always, it's recommended to encrypt the credentials with our
```

Comment:
Couple of comments here:

* better to cut to the chase on line 499 "To customize the message"...
* please use triple backtick code sections as they have syntax highlighting

```
```json
```